cmd/testscript/testdata/continue.txt

```diff
@@ -0,0 +1,14 @@
+# should support -continue
+unquote file.txt
+
+# Running with continue, the testscript command itself
+# should fail, but we should see the results of executing
+# both commands.
+! testscript -continue file.txt
+stdout 'exec false'
```

Comment:
Alternatively, rather than relying on the presence of the `false` and `true` executables, how about just defining an `error` command that prints an error and returns a non-zero status. Then you avoid that dependency and also get to check that the failure output contains the output from failing commands.

Reply:
In fact, no need to define a new command - maybe just run a `grep` command that's expected to fail instead?

Reply:
nit: would also be nice to run the test script without continue, to double check that the behavior does change with the flag on.

Reply:
That's a really good point re `false` and `true`. I'll fix that up.

Reply:
Fixed.